src/util.cpp

```diff
@@ -396,7 +403,13 @@ bool GetBoolArg(const std::string& strArg, bool fDefault)
     {
         if (mapArgs[strArg].empty())
             return true;
-        return (atoi(mapArgs[strArg]) != 0);
+        int b;
+        if (ParseInt32(mapArgs[strArg], &b)) {
+            return b != 0;
+        }
+        // Perhaps fDefault should be returned here, but this mimics the
+        // (previously used) atoi64() behavior
+        return false;
```

Comment:
Changing this to return `fDefault` (instead of `false`) did not cause any tests to fail, but I was nervous, so I opted to stay more compatible (the old code returns `false` if the string doesn't parse, because `atoi` returns zero). Let me know if you think otherwise.